recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,86 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "nnpops" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 64eb48e4a2a642f5e951fe368c46bd41402ce68a89f2f56189728b4f551cbe9d
+
+build:
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
```

Comment:
why are you only building cuda? this support pure CPU as well which is typically easier to build